prbt_hardware_support/src/pilz_modbus_client_node.cpp

```diff
@@ -104,15 +104,19 @@ int main(int argc, char **argv)
   nh.param<std::string>(PARAM_MODBUS_TOPIC_NAME_STR, modbus_topic_name,
                         TOPIC_MODBUS_READ);
 
+  std::string modbus_write_topic_name;
+  nh.param<std::string>(PARAM_MODBUS_WRITE_TOPIC_NAME_STR, modbus_write_topic_name,
+                        TOPIC_MODBUS_WRITE);
+
 
   // LCOV_EXCL_STOP
 
-  prbt_hardware_support::PilzModbusReadClient modbus_client(nh,
+  prbt_hardware_support::PilzModbusClient modbus_client(nh,
                                                             static_cast<uint32_t>(num_registers_to_read),
                                                             static_cast<uint32_t>(index_of_first_register),
                                                             std::unique_ptr<LibModbusClient>(new LibModbusClient()),
                                                             static_cast<unsigned int>(response_timeout_ms),
-                                                            modbus_topic_name);
+                                                            modbus_topic_name, modbus_write_topic_name);
 
   ROS_DEBUG_STREAM("Modbus client IP: " << ip << " | Port: " << port);
   ROS_DEBUG_STREAM("Number of registers to read: " << num_registers_to_read
```

Comment:
are they only to read? or also to write?
if 1) add debug output for write
if 2) change variable name